tasks/libs/common/gomodules.py

```diff
@@ -305,14 +305,15 @@ def dependency_path(self, agent_version):
 AGENT_MODULE_PATH_PREFIX = "github.com/DataDog/datadog-agent/"
 
 
-@lru_cache
```

Comment:
Yes, I removed it since now the modules within the worktree cannot be cached (the worktree has always the same path but the modules can be changed if we change the branch)